glue_jupyter/bqplot/common/tools.py

```diff
@@ -178,6 +178,73 @@ def activate(self):
         super().activate()
 
 
+@viewer_tool
+class BqplotPolygonMode(BqplotSelectionTool):
+
+    icon = os.path.join(ICONS_DIR, 'glue_lasso')
```

Comment:
```suggestion
    icon = 'glue_lasso'
```

should work as it's a standard icon in glue-core

Reply:
Can you confirm if you were trying this in Jupyter Lab or Jupyter Notebook? I was developing in Jupyter Lab, in Notebook I get what seems like your problem. Just want to confirm that it's the same thing...

Reply:
I think I was using notebook